src/command_modules/azure-cli-find/setup.py

```diff
@@ -32,7 +32,7 @@
 
 DEPENDENCIES = [
     'azure-cli-core',
-    'whoosh'
+    'colorama~=0.3.7'
```

Comment:
Don't add this. The CLI core already specifies a colorama dependency and this could conflict with that.